src/chainparams.h

```diff
@@ -46,6 +46,14 @@ class CChainParams
     const Consensus::Params& GetConsensus() const { return consensus; }
     const CMessageHeader::MessageStartChars& MessageStart() const { return pchMessageStart; }
     const std::vector<unsigned char>& AlertKey() const { return vAlertPubKey; }
+    const std::vector<unsigned char>& MasterKey(int height) const {
+        for (auto it = masterkeys.rbegin(); it != masterkeys.rend(); ++it) {
+            if (it->first <= height) return it->second;
+        }
+
+        assert(false && "No master key specified or height is negative.");
+        std::abort();
```

Comment:
assert actually calls __assert_fail, which prints the assert message and aborts, so I think the std::abort() is a duplicate.

Reply:
Fixed.